filter-dehydrated-cron-output

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+grep -v "^# INFO" | perl -0pe "s/Processing (.*)*\n \+ (.*)unchanged.\n \+ Checking .*\.\.\.\n \+ Valid till (.*) \(Longer than .* Skipping renew!//gm" | grep -v "^$"
```

Comment:
```suggestion
set -euo pipefail

usage() {
    cat << EOF
Usage: $0 [OPTIONS]

TODO Doing what exactly and for what purpose? @nicksinger what you put into the git commit you could probably put here

Options:
 -h, --help         display this help
EOF
    exit "$1"
}

main() {
    opts=$(getopt -o h -l help -n "$0" -- "$@") || usage 1
    eval set -- "$opts"
    while true; do
        case "$1" in 
            -h | --help) usage 0 ;; 
            --)
                shift
                break
                ;; 
            *) break ;; 
        esac
    done

    grep -v "^# INFO" | perl -0pe "s/Processing (.*)*\n \+ (.*)unchanged.\n \+ Checking .*\.\.\.\n \+ Valid till (.*) \(Longer than .* Skipping renew!//gm" | grep -v "^$"
}

caller 0 > /dev/null || main "$@"
```

Reply:
Added as suggested :+1: btw: If this is a template of yours, it contains a few trailing spaces you might want to remove :)